UnityProject/Assets/Scripts/TutorialScript/Lang_Bot.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Xml;
+using UnityEngine;
+
+
+public class Lang_Bot
```

Comment:
classes should all use ``PascalCase`` for their names. You should also add a namespace for this class.